### ISSUE TITLE
Add check if .PKG is signed 

### ISSUE THREAD

**Issue by allexb** [msintuneappsdk/intune-app-wrapping-tool-mac#14]
Add check if .PKG is signed because of download problem to client

**Comment by Hipster74** [comment]
Yes, how about sanity checking the .pkg before allowing it to be converted to .cmmac.

**Comment by allexb** [comment]
It is used in enterprise but no answer from the MS developers.
